### PR TITLE
Add a volume level rebalance status field

### PR DESCRIPTION
**Pull request by nnDarshan**
tendrl-bug-id: Tendrl/gluster-integration#323
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@shtripat @nthomas-redhat @r0h4n PLease review

File: tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def run(self):
                 )
             ).value
             if rebal_status is not None:
-                if rebal_status == "in progress":
+                if rebal_status == "in progress" or rebal_status == "in_progress":
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py]: if rebal_status in ["in progress", "in_progress"]

Comment (nnDarshan) [inline tendrl/gluster_integration/objects/volume/atoms/rebalance_running/__init__.py]: ack

**Comment by nnDarshan** [comment]
@r0h4n @nthomas-redhat @shtripat The change has been verified from backend and WIP UI patch.
@gnehapk has posted patches in UI for taking this care of this, should be merged soon

**Pull request merged**